src/style.css

```diff
@@ -0,0 +1,145 @@
+* {
+  margin: 0 ;
+  padding: 0 ;
+  border: 0 ;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box ;
+}
+
+a:focus,
+a:active {
+  outline: none ;
+}
+
+a:visited,
+a:hover {
+  text-decoration: none ;
+}
```

Comment:
don't style tags, use classes, but i'm not sure you really need this part of css